go/pkg/client/cas.go

```diff
@@ -565,6 +574,10 @@ func (c *Client) WriteProto(ctx context.Context, msg proto.Message) (digest.Dige
 func (c *Client) WriteBlob(ctx context.Context, blob []byte) (digest.Digest, error) {
 	ue := uploadinfo.EntryFromBlob(blob)
 	dg := ue.Digest
+	if dg.IsEmpty() {
+		LogContextInfof(ctx, log.Level(2), "Skipping upload of empty blob")
```

Comment:
Include blob digest? For all the logs.

Reply:
It's the empty digest, though... But sure, will do.